opefeature-web-provider/src/demo.ts

```diff
@@ -0,0 +1,28 @@
+import { OpenFeature, ProviderEvents } from "@openfeature/web-sdk"
```

Comment:
Could we align This demo with https://github.com/Flagsmith/flagsmith-js-examples/blob/main/javascript-vanilla/main.js in terms of feature parity and html? I think it'll make this a lot easier to follow and identify if anything is missing.

Reply:
Here the features like traits, login, logout from flagSmith are not supported in open-feature. But we can try to find a work around that would enable us to do implement these features. Let me know your thoughts on this.

Reply:
Yes it's very important that we get feature parity, otherwise we wouldn't want to recommend it.

Reply:
Reading the docs, this is done via contexts. Contexts allow you to pass a "targetingKey"(identity) as well as other key value pairs (traits)